rpc/dcrwallet.go

```diff
@@ -155,6 +143,31 @@ func (w *WalletConnect) Clients() ([]*WalletRPC, []string) {
 	return walletClients, failedConnections
 }
 
+// version uses version RPC to retrieve the API version of dcrwallet.
+func (c *WalletRPC) version() (*semver, error) {
+	var verMap map[string]dcrdtypes.VersionResult
+	err := c.Call(context.TODO(), "version", &verMap)
```

Comment:
instead of introducing even more TODO, why not write this with context support and pass in TODO from the caller, if that is still needed?